src/qt/rpcconsole.cpp

```diff
@@ -897,57 +897,71 @@ void RPCConsole::setMempoolSize(long numberOfTxs, size_t dynUsage)
 
 void RPCConsole::on_lineEdit_returnPressed()
 {
-    QString cmd = ui->lineEdit->text();
+    QString cmd = ui->lineEdit->text().trimmed();
```

Comment:
nit

we should document that we are using this function in the commit message for proper documentation

Reply:
This could be addressed in a follow up.

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/123#issuecomment-851674465).